src/DynamoCoreWpf/ViewModels/Watch3D/IWatch3DViewModel.cs

```diff
@@ -45,7 +45,18 @@ public struct RenderDescriptions
         /// </summary>
         public const string ManipulatorPlane  = "E75B2B0E31F1";
     }
-
+    /// <summary>
+    /// interface that defines properties of the Watch3dView required for preference serialization.
+    /// this interface should be merged with IWatch3dViewModel at Dynamo 2.0
+    /// </summary>
+    public interface IWatchPreferenceProperties
+    {
+        /// <summary>
+        /// Represents the name of current IWatch3DViewModel which will be saved in preference settings
+        /// </summary>
+        string PreferenceWatchName { get; }
```

Comment:
I guess we should also keep a list of these changes in the API changes doc potentially.